content/cpp/concepts/math-functions/terms/atan/atan.md

```diff
@@ -21,19 +21,35 @@ atan(n)
 
```

Comment:
Can we write 1-3 sentences here that explains the domain/range of the `n` parameter along with any edge cases (e.g., pos/neg infinity).